src/main/java/com/example/main/SampleApplication.java

```diff
@@ -47,17 +63,60 @@ public class SampleApplication implements CommandLineRunner {
     @Autowired
     private ApplicationContext applicationContext;
 
+    @Autowired
+    private CosmosTemplate cosmosTemplate;
+
+    @Autowired
+    private ReactiveCosmosTemplate reactiveCosmosTemplate;
+
     public static void main(String[] args) {
         SpringApplication.run(SampleApplication.class, args);
     }
 
     public void run(String... var1) {
 
-        final User1 testUser11 = new User1("testId1", "testFirstName", "testLastName1");
-        final User1 testUser12 = new User1("testId2", "testFirstName", "testLastName2");
+        ChangeFeedProcessorOptions changeFeedProcessorOptions = new ChangeFeedProcessorOptions();
+        changeFeedProcessorOptions.setStartFromBeginning(true);
+        changeFeedProcessorOptions.setLeasePrefix("SOME_PREFIX");
+        changeFeedProcessorOptions.setFeedPollDelay(Duration.ofMillis(0L).plusSeconds(5));
+
+        CosmosAsyncClient cac = primaryDataSourceConfiguration
```

Comment:
Is customer also building the client separately?

Reply:
They haven't posted how they are doing it, there isn't a client created anywhere to use so you have to create one

Reply:
The client is always created by spring application. Without client creation, nothing will work. You should get the client from applicationContext like we do in our spring data cosmos project in some test cases.

Reply:
Done.